tests/templates/test_subroutines/test_amplitude_amplification.py

```diff
@@ -0,0 +1,336 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Tests for the AmplitudeAmplification template.
+"""
+
+import pytest
+import numpy as np
+import pennylane as qml
+from pennylane.templates.subroutines.amplitude_amplification import _get_fixed_point_angles
```

Comment:
Surprised pylint doesn't complain about this. Was this file added to the list that is covered by pylint? I vaguely remember having to do this manually.

Reply:
I have not added this file anywhere 🤔

Reply:
Never mind! Spoke with Matt, this is not needed anymore.